stdlib/ballerina-builtin/src/main/ballerina/ballerina/io/print.bal

```diff
@@ -23,3 +23,9 @@ public native function print (any a);
 @Description { value:"Prints an any value to the STDOUT in a new line"}
 @Param { value:"a: any value to be printed" }
 public native function println (any a);
+
+@Description { value:"Returns a formatted string using the specified format string and arguments" }
+@Param { value:"format: Format specifier" }
+@Param { value:"args: Arguments to be formatted, should match number of args in format specifier" }
+@Return { value:"string: Formatted string" }
```

Comment:
Only the variable name is expected before colon. Shall we remove "string:" part which is the variable type?